hazelcast/src/main/java/com/hazelcast/internal/ascii/rest/HttpCommandProcessor.java

```diff
@@ -78,8 +78,8 @@
     public static final String URI_CP_SUBSYSTEM_BASE_URL = "/hazelcast/rest/cp-subsystem";
     public static final String URI_RESET_CP_SUBSYSTEM_URL = URI_CP_SUBSYSTEM_BASE_URL + "/reset";
     public static final String URI_CP_GROUPS_URL = URI_CP_SUBSYSTEM_BASE_URL + "/groups";
-    public static final String URI_CP_SESSIONS_SUFFIX = "/sessions";
-    public static final String URI_REMOVE_SUFFIX = "/remove";
+    public static final String URI_CP_SESSIONS_SUFFIX = URI_CP_SUBSYSTEM_BASE_URL + "/sessions";
```

Comment:
@metanet was this intended or a mistake? If the former, please let me know so I can revert.

Reply:
I think this was intended as it's a suffix that can occur at the end of some URI, for instance see [here](https://github.com/hazelcast/hazelcast/blob/ff3ce46430957433f047395a8e927144c9308b8d/hazelcast/src/test/java/com/hazelcast/internal/ascii/HTTPCommunicator.java#L369-L372) and [here](https://github.com/hazelcast/hazelcast/blob/ff3ce46430957433f047395a8e927144c9308b8d/hazelcast/src/test/java/com/hazelcast/internal/ascii/HTTPCommunicator.java#L374-L378).